website/content/en/docs/ansible/quickstart.md

```diff
@@ -377,4 +377,4 @@ For more information, refer [cli][addcli] doc.
 [kubectl-tool]:https://kubernetes.io/docs/tasks/tools/install-kubectl/
 [addcli]: /docs/cli/operator-sdk_add_api
 <!-- TODO: update this link to the non-legacy doc once the ansible plugin is publicly available -->
-[quickstart-bundle]: /docs/olm-integration/legacy/quickstart-bundle
+[quickstart-bundle]:https://v0-19-x.sdk.operatorframework.io/docs/olm-integration/legacy/quickstart-bundle/
```

Comment:
I'm assuming this is fully qualified because legacy only lives in 0.19 now.

Reply:
Once the ansible operator is updated, we'll probably update this link again to point to the new KB-based bundle quickstart.